libcodechecker/cmd/cmd_line_client.py

```diff
@@ -321,11 +318,7 @@ def handle_list_results(args):
                 rows.append(
                     (checked_file, res.checkerId, sev, res.checkerMsg))
 
-        if args.output_format == 'csv':
-            writer = csv.writer(sys.stdout)
-            writer.writerows(rows)
-        else:
-            print_table(rows)
+        print(twodim_to_str(args.output_format, header, rows))
```

Comment:
`ValueError` exceptions should be handled if `twodim_to_str` is called, what if the user makes a typo in the format?